openage/convert/terrain_convert.pyx

```diff
@@ -8,66 +8,89 @@ Transforms the dimetric tiles converted from SLP files to cartesian format.
 
 import math
 import argparse
-from PIL import Image
 from .slp import SLP
 from .texture import Texture
+from PIL import Image
+import numpy as np
 
 class Terrain_convert:
-
+    
+    # Method to simply put the tile on the big merge img. Something is wrong on this method because the result is transparent.
+    # TO FIX.
+    @staticmethod
+    def replace(overlayer, overlayed, overlayer_x, overlayer_y, overlayed_x, overlayed_y, slp_width, slp_height):
+        to_ret = overlayed
+        xr = overlayer_x
+        xd = overlayed_x
+        yr = overlayer_y
+        yd = overlayed_y
```

Comment:
This needs more commenting :D For example, what is this function supposed to do, what are `overlayer` and `overlayed` supposed to be. It is not clear from the code.

Also, it would help if you used more descriptive variable and method names for easier understanding.

Reply:
Of course, I'll add specs on the next commit, but basically, this function is used to paste a tile on the final RGBA array of the terrain texture. overlayer are tiles (and overlayed is the merge_img), because you overwrite a subarray of the terrain array (merge_img) with a tile.
I use nested because I haven't found something more optimal. I simply paste pixel per pixel.